litellm/litellm-config.yaml

```
@@ -0,0 +1,37 @@
+      output_cost_per_token: 0.000015
+  
+  # - model_name: Meta_Llama_31_8b_it
+  #   litellm_params:
+  #     model: openai/meta-llama-31-8b-it
+  #     api_base: os.environ/BASE_URL
+  #     api_key: os.environ/LLAMA31_API_KEY
+  #     max_tokens: 1000
+  #     input_cost_per_token: 0.000003
+  #     output_cost_per_token: 0.000003
+
+general_settings: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)